spaghetto/src/Spago/Config.purs

```diff
@@ -290,12 +292,43 @@ type WorkspacePackage =
   , hasTests :: Boolean
   }
 
+workspacePackageCodec :: JsonCodec WorkspacePackage
+workspacePackageCodec = CAR.object "WorkspacePackage"
+  { path: CA.string
+  , package: packageConfigCodec
+  , doc: yamlDocCodec
+  , hasTests: CA.boolean
+  }
+
+yamlDocCodec :: JsonCodec (YamlDoc Config)
+yamlDocCodec = CA.codec' decode encode
+  where
+  -- TODO: implementation of encode
+  encode _x = CA.encode CA.null unit
+
+  -- TODO: implementation of decode if needed
+  decode _json = Left MissingValue
```

Comment:
:information_source: As I wrote somewhere else, my assumption is that encoding this - at least in the dependencies - would make sense. Open to other suggestions here (noop, different type).